config/recipe-hyprland.yml

```diff
@@ -28,6 +28,37 @@ modules:
       - hyprland
       - waybar
       - xdg-desktop-portal-hyprland
+      - xorg-x11-server-Xwayland
+
+      # wayland utilities
+      - swaybg
+      - swayidle
+      - swaylock
+      - wlr-randr
+      - wl-sunset
+
+      # essentials for usability
+      - pipewire
+      - wireplumber
+      - xdg-desktop-portal-hyprland
+      - xdg-user-dirs
+      - polkit-kde
+      - qt5-qtwayland
+      - qt6-qtwayland
+
+      # improved desktop experience
+      - bluez
+      - bluez-tools
+      - blueman
+      - network-manager-applet
+      - pavucontrol
+
+      # additional GUI applications
+      - thunar
+      - lxappearance
```

Comment:
what is this used for?

Reply:
lxappearance? Used for managing GTK themes, although I think it is only applying to GTK 2, but not 3 or 4. I was looking int nwg-look (https://github.com/nwg-piotr/nwg-look), but it currently isn't available on Fedora, yet.

Reply:
got it, let's keep it then